examples/vae.py

```diff
@@ -133,7 +133,7 @@ def reconstruct_img(epoch, rng_key):
 if __name__ == '__main__':
     assert numpyro.__version__.startswith('0.2.3')
     parser = argparse.ArgumentParser(description="parse args")
-    parser.add_argument('-n', '--num-epochs', default=20, type=int, help='number of training epochs')
+    parser.add_argument('-n', '--num-epochs', default=15, type=int, help='number of training epochs')
```

Comment:
with 20 epochs, loss returns nan in my system.

Reply:
Hmm..this is probably worth investigating separately.